interfaces/azure/v0/README.md

```diff
@@ -0,0 +1,65 @@
+# `azure`
```

Comment:
I think this name should be more specific, perhaps `azure-storage` or similar?

Reply:
Fixed in 949c91b.